tests/test_module_clustrmsd.py

```diff
@@ -352,3 +353,20 @@ def test_iterate_min_population():
     exp_cluster_arr = np.array([1, 1, -1, -1, -1])
     assert obs_min_population == 2
     assert (obs_cluster_arr == exp_cluster_arr).all()
+
+
+def test_order_clusters():
+    """Test order_clusters function."""
+    cluster_arr = np.array([1, 1, 2, 3, 4])
+    obs_clusters, obs_cluster_arr = order_clusters(cluster_arr)
+    exp_clusters = [1, 2, 3, 4]
+    exp_cluster_arr = np.array([1, 1, 2, 3, 4])
+    assert obs_clusters == exp_clusters
+    assert (obs_cluster_arr == exp_cluster_arr).all()
+    # now with a less trivial cluster_arr
+    cluster_arr = np.array([3, 3, 2, 4, 3, 1, 3, 3, 1, 3, 4])
```

Comment:
I have the feeling that cluster id `-1` is not tested

Reply:
good catch! I corrected the function and added the test, should be fine now